docker_registry/images.py

```diff
@@ -60,6 +62,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def valid_image_id(f):
```

Comment:
Does this maybe belong in toolkit? Keep only view functions in images.py as much as possible.